부산대 FE_안형찬: 1단계 - 프로젝트 세팅

hyeongchanan opened the pull request:
안녕하세요 부산대학교 FE 트랙 수강중인 안형찬입니다.
제출이 늦어져서 정말 죄송합니다.
1.README.md작성

2.불필요한 파일 코드 정리
-svg파일 삭제 App.tsx 수정 했습니다

3.절대경로로 Import 할 수 있게 alias를 설정
-tsconfig.json, vite.config.ts 에 alias설정을 추가했습니다.

4.Prettier 추가
-패키지 설치및, .prettier 파일을 추가했습니다

5.emotion 스타일 라이브러리를 추가, reset css를 적용

emotion라이브러리를 설치와 reset.ts, global.tsx 를 추가했습니다.
6.기본 폰트로 Pretendard를 적용

reset.ts 에 코드를 추가했습니다.

hyeongchanan committed:
README.md 업데이트

프로젝트 설명 추가
구현 기능 목록 추가

hyeongchanan committed:
불피요한 파일, 코드정리

.svg 파일 삭제
app.tsx 초기화

hyeongchanan committed:
Merge branch 'upstream/anhyeongchan' of https://github.com/hyeongchanan/react-gift-login into upstream/anhyeongchan

hyeongchanan committed:
prettier 설치 및 설정

pakage.json 수정
.prettierrc 추가

hyeongchanan committed:
프로젝트 세팅 완료

emotion 스타일 라이브러리 추가, recet css 적용
기본 폰트로 Pretendard 설정

ed-jinyoung-park reviewed (changes requested):
안녕하세요 형찬님 반갑습니다 이번에 리뷰어를 맡게 된 박진영입니다 🙂 잘 부탁드려요

우선 1단계 프로젝트 세팅 하시느라 고생많으셨습니다.

요구사항들을 대부분 다 해결해주셨는데요. 폰트 적용이 제대로 되지 않은 이슈가 하나 있어 직접 확인해보시고 수정해보시는 게 좋을 것 같아 request change를 드립니다. 코멘트를 확인해 보시면 금방 수정 해보실 수 있을 것 같아요!

(다른분들 PR에서도 남겼지만) 과제 진행하면서 질문이나 고민이 되었던 부분이 있으시면 PR description이나 코멘트 혹은 디스코드 DM을 통해 편하게 남겨주시면 좋을 것 같습니다.

다시 한번 고생 많으셨습니다 앞으로도 화이팅입니다👏

hyeongchanan commented:
감사합니다! 말씀해주신부분 검토해서 다시 올리겠습니다!

hyeongchanan committed:
1차 리뷰후 수정

1.폰트 적용방식을 CDN -> CSS 방식으로 변경  index.html 수정 reset.ts 수정
2.prettier vscode extion 으로 설치, 기존 pakage.json 코드 삭제

ed-jinyoung-park reviewed (approved):
형찬님 고생하셨습니다 잘 수정해주신 것 같아 다음 단계로 넘어갈게요!

한가지 제가 놓친게 있는데 커밋 메시지를 [AngularJS Git Commit Message Conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)[AngularJS Git Commit Message Conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)
에 맞춰서 작성해주시는 것이 요구사항에 있어서 2단계부터는 해당 컨벤션에 맞춰서 작성하시는 연습을 해주시면 좋겠습니다. 그럼 다음 단계에서 봬요!
